src/modules/new-request-form/data-types/Organization.ts

```diff
@@ -0,0 +1,5 @@
+export interface Organization {
+  id: number;
+  name: string;
+  default: true;
```

Comment:
We don't have a default property, right? In this case I would remove it from the type

Reply:
Right, we changed that. I will remove it